docker/.env

```diff
@@ -1,21 +1,22 @@
 # This is the port the vue.js frontend listens on. You can
```

Comment:
There are a few comments here and there that still reference `vue`

Reply:
Ah, so the frontend is still written in vue.js (that's where all those `.vue` extensions in the source directory come in).

The difference in this PR is that the build/run/serve steps happen with a different/more modern/actually supported tool (`vue-cli` -> `vite`). If you're curious enough, you can read through this: https://www.codemag.com/Article/2109071/Introducing-Vite-A-Better-Vue-CLI

Reply:
Thanks for the link, I figured it was something like that (I did a google search after submitting my review and skimmed the article titles)